evalml/tests/component_tests/test_one_hot_encoder.py

```diff
@@ -310,7 +321,7 @@ def test_numpy_input():
     encoder = OneHotEncoder()
     encoder.fit(X)
     X_t = encoder.transform(X)
-    assert pd.DataFrame(X).equals(X_t)
+    pd.testing.assert_frame_equal(pd.DataFrame(X), X_t, check_dtype=False)
```

Comment:
As mentioned in standup, windows will use int32 as opposed to int64, so not checking for dtype equality. https://app.circleci.com/pipelines/github/alteryx/evalml/7300/workflows/0d609f82-10ba-4cf8-84b2-01e9ecf7864f/jobs/81615

Reply:
Hey I'm glad `assert_frame_equal` handles this case :)

Reply:
Yes, agreed! 😂